app/src/main/kotlin/com/looker/droidify/model/Repository.kt

```diff
@@ -397,6 +397,24 @@ data class Repository(
                     "Cromite is a Chromium fork based on Bromite with " +
                     "built-in support for ad blocking and an eye for privacy.",
                 fingerprint = "49F37E74DEE483DCA2B991334FB5A0200787430D0B5F9A783DD5F13695E9517B"
+            ),
+            defaultRepository(
+                address = "https://brave-browser-apk-release.s3.brave.com/fdroid/repo",
+                name = "Brave Release",
```

Comment:
I think better to rename it as "Brave Browser" as it's the stable release version.